docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/FeatureSelection/SelectFeaturesBasedOnCount.cs

```diff
@@ -0,0 +1,117 @@
+            // It will be stored in the same path as the executable
+            string dataFilePath = SamplesUtils.DatasetUtils.DownloadBreastCancerDataset();
+
+            // Data Preview
+            //    1. Label							0=benign, 1=malignant
+            //    2. Clump Thickness               1 - 10
+            //    3. Uniformity of Cell Size       1 - 10
+            //    4. Uniformity of Cell Shape      1 - 10
+            //    5. Marginal Adhesion             1 - 10
+            //    6. Single Epithelial Cell Size   1 - 10
+            //    7. Bare Nuclei                   1 - 10
+            //    8. Bland Chromatin               1 - 10
+            //    9. Normal Nucleoli               1 - 10
+            //   10. Mitoses                       1 - 10
+
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // First, we define the loader: specify the data columns and where to find them in the text file. Notice that we combine entries from
+            // all the feature columns into entries of a vector of a single column named "Features".
+            var loader = mlContext.Data.CreateTextLoader(
+                columns: new[]
+                    {
+                        new TextLoader.Column("Label", DataKind.Boolean, 0),
+                        new TextLoader.Column("GroupA", DataKind.Single, new [] { new TextLoader.Range(1, 3) }),
+                        new TextLoader.Column("GroupB", DataKind.Single, new [] { new TextLoader.Range(4, 6) }),
+                        new TextLoader.Column("GroupC", DataKind.Single, new [] { new TextLoader.Range(7, 9) }),
+                    },
+                hasHeader: true
+            );
+
+            // Then, we use the loader to load the data as an IDataView.
+            var data = loader.Load(dataFilePath);
+
+            Console.WriteLine("Contents of column 'GroupB'");
+            PrintDataColumn(data, "GroupB");
+            // 5 7 10
+            // 1 2 2
+            // 1 3 4
+            // 3 2 1
+            
+            // Second, we define the transformations that we apply on the data. Remember that an Estimator does not transform data
+            // directly, but it needs to be trained on data using .Fit(), and it will output a Transformer, which can transform data.
+
+            // In this example we define a CountFeatureSelectingEstimator, that selects slots in a feature vector that have more non-default 
+            // values than the specified count. This transformation can be used to remove slots with too many missing values.
+            var pipeline = mlContext.Transforms.FeatureSelection.SelectFeaturesBasedOnCount(
+                outputColumnName: "FeaturesSelectedGroupB", inputColumnName: "GroupB", count: 695);
```

Comment:
>695 [](start = 94, length = 3)

Where this number coming from?

Reply:
comment in line 29 should clarify this now, also the in-memory example should make it more intuitive

---
In reply to: [271911491](https://github.com/dotnet/machinelearning/pull/3184#discussion_r271911491) [](ancestors = 271911491)